mutation-testing/check-packages-and-shards/action.yml

```diff
@@ -0,0 +1,123 @@
+name: Check Packages and Shards
+
+outputs:
+  run_big_packages:
+    value: ${{ steps.check_packages_and_shards.outputs.run_big_packages }}
+  big_packages_with_shards:
+    value: ${{ steps.check_packages_and_shards.outputs.big_packages_with_shards }}
+  run_small_packages:
+    value: ${{ steps.check_packages_and_shards.outputs.run_small_packages }}
+  small_packages_with_shards:
+    value: ${{ steps.check_packages_and_shards.outputs.small_packages_with_shards }}
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Checkout stacks-core repo
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Install cargo-mutants
```

Comment:
prefer to use an existing action here, i.e. https://github.com/stacks-network/actions/blob/main/stacks-core/cache/cargo/action.yml#L75-L82

vs installing via shell